lib/src/block_parser.dart

```diff
@@ -597,24 +599,39 @@ abstract class ListSyntax extends BlockSyntax {
       return match != null;
     }
 
+    int expandedTabLength(String input) {
```

Comment:
Needs to be inline? Could be class static? Top-level?

Reply:
You're right. I like static private.